pilot/pkg/model/authentication.go

```diff
@@ -69,6 +69,11 @@ func ConstructSdsSecretConfig(serviceAccount string, sdsUdsPath string) *auth.Sd
 								GoogleGrpc: &core.GrpcService_GoogleGrpc{
 									TargetUri:  sdsUdsPath,
 									StatPrefix: SDSStatPrefix,
+									ChannelCredentials: &core.GrpcService_GoogleGrpc_ChannelCredentials{
```

Comment:
what is the GoogleGrpc service?

Reply:
gRPC https://grpc.io/docs/guides/index.html, the user case here is sds client(envoy) communicates with sds server(node agent) through gRPC.